results/cmd/api/main.go

```diff
@@ -81,3 +106,24 @@ func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest
 	}
 	return taskrunFromClient, nil
 }
+
+func (s *server) DeleteTaskRun(ctx context.Context, req *pb.DeleteTaskRunRequest) (*empty.Empty, error) {
+	database := s.db
+	statement, err := database.Prepare("DELETE FROM taskrun WHERE uid = ?")
+	if err != nil {
+		log.Fatalf("failed to create delete statement: %v", err)
+	}
+	results, err := statement.Exec(req.GetUid())
+	if err != nil {
+		log.Fatalf("failed to execute delete statement: %v", err)
+	}
+	affect, err := results.RowsAffected()
+	if err != nil {
+		log.Fatalf("failed to retrieve results: %v", err)
+	}
+	fmt.Println(affect)
+	if affect == 0 {
+		return &(empty.Empty{}), errors.New("Row Not Found: did not find a row with specified condition in database")
```

Comment:
minor nit: row not found: instead of Row Not Found

Reply:
Done